content/rs/clusters/monitoring/prometheus-integration.md

```diff
@@ -105,30 +104,41 @@ This is for development testing only (running in Docker).
                     - prometheus-server:prometheus
         ```
 
-    1. To start the containers, run: `docker-compose up -d`
+    1. To start the containers, run:
+
+    ```sh
+    $ docker compose up -d
+    ```
```

Comment:
I think you might need to indent this code block so that the steps don't start over from `a`. Currently, the steps go `a, b, a, b, c`.
```suggestion
        ```sh
        $ docker compose up -d
        ```
```